components/account/ManageSessions.tsx

```diff
@@ -35,8 +36,8 @@ const ManageSessions = () => {
       });
 
       if (!response.ok) {
-        const json = await response.json();
-        throw new Error(json.error.message);
+        const json = (await response.json()) as ApiResponse;
+        toast.error(t(json?.error?.message || 'Something went wrong'));
```

Comment:
If error is not thrown here the code bubbles down, need to return here.

Reply:
@deepakprabhakara  ok am lost is the way shown above the correct way or relace the green line 40 toast.error with throw new Error to be caught by the catch

Reply:
```suggestion
        throw new Error(t(json?.error?.message || 'Something went wrong'));
```